gensim/models/lsimodel.py

```diff
@@ -482,14 +483,28 @@ def add_documents(self, corpus, chunksize=None, decay=None):
         If the distributed mode is on, each chunk is sent to a different worker/computer.
 
         """
+        def is_empty(corpus):
```

Comment:
Let's put this in `utils.py`. Rather than an inline (anonymous) function.

Reply:
@mark-todd can you move this function to `utils.py` please?

Reply:
Yep tis done - sorry about the delay